modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,44 @@
+//tag::intro1[]
+== TLS Support on Flex Gateway
+
+Flex can support a range of TLS Ciphers from TLS 1.1 to TLS 1.3. You cannot support both TLS 1.1 and TLS 1.3 without also supporting TLS 1.2. The default minimum and maximum values for Flex Gateway connected mode are both 1.3. 
+
+You cannot customize the list of TLS 1.3 Ciphers.
+
+IMPORTANT: Ensure that your API upstream supports the selected ciphers and versions.
+
+//end::intro1[]
+
+//tag::addContext[]
+== Flex Gateway Supported Ciphers
+
+Flex Gateway supports the following TLS Ciphers in connected and local mode:
```

Comment:
Note that these are ciphers supported for TLS versions lower than 1.3. I would clarify that here